models/llama/example_text_completion.py

```diff
@@ -0,0 +1,64 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+# This software may be used and distributed in accordance with the terms of the Llama 3 Community License Agreement.
+
+from typing import List
+
+import fire
+
+from .llama import Llama
```

Comment:
This relative import causes problems for me, and upstream has
```
from llama import Llama
```

Reply:
can you show the error message you got?

Reply:
```
Traceback (most recent call last):
  File "/data/users/bertrand/kernels/models/llama/./example_text_completion.py", line 8, in <module>
    from .llama import Llama
ImportError: attempted relative import with no known parent package
```